www/front_src/src/Resources/Details/Body/tabs/Details/StateCard.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { Card, CardContent, Grid, Typography } from '@material-ui/core';
+
+interface Props {
+  title: string;
+  contentLines: Array<string>;
+  chip: JSX.Element;
+}
+
+const Line = (line): JSX.Element => (
+  <Typography key={line} variant="body2" component="p">
+    {line}
+  </Typography>
+);
+
+const StateCard = ({ title, contentLines, chip }: Props): JSX.Element => {
+  return (
+    <Card>
+      <CardContent>
+        <Grid container spacing={4} alignItems="center">
```

Comment:
did not check the result, but a spacing of 24px is not too big ?

Reply:
It's consistent with the given wireframes.

We can always change it if requested during the feedback phase